File: CourseProjects/CourseProjects/VisualStudioProjects/Agile/IntroducingNUnit/NUnitWithVB/BasicMath.vb
Public Class BasicMath
    Public Sub New(ByVal no1 As Integer, ByVal no2 As Integer)
        Me.no1 = no1
        Me.no2 = no2
    End Sub
    Public Function Add() As Integer
        Return no1 + no2
    End Function
    Public Function Subtract() As Integer
        Return no1 - no2
    End Function
    Public Function Multiply() As Integer
        Return no1 * no2
    End Function
    Private no1 As Integer
    Private no2 As Integer
End Class

File: CourseProjects/CourseProjects/VisualStudioProjects/Agile/IntroducingNUnit/Backup/NUnitWithCS/BasicMath.cs
using System;

namespace TestBasicMath {
	public class BasicMath {
		public BasicMath(int no1, int no2) : base() {
			this.no1 = no1;
			this.no2 = no2;
		}
		public int add() {
			return no1 + no2;
		}
		public int subtract() {
			return no1 - no2;
		}
		public int multiply() {
			return no1*no2;
		}
		private int no1;
		private int no2;
	}
}